public/assets/images/icons/icon-linkedin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="3 3 27 27" fill="#FFFFFF" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Random comment that doesn't matter, the other icons besides for the eventbrite one have the height and width specified as 25. but if you change the width and height it doesn't do anything haha. Also in the actual app it looks awesome so this cool with me!

Reply:
Since you're 100% right about it not doing anything in the app I'll make this change too for consistency!!